code/modules/reagents/reagent_containers/food/drinks.dm

```diff
@@ -1108,6 +1108,21 @@
 	src.pixel_x = rand(-10, 10) * PIXEL_MULTIPLIER
 	src.pixel_y = rand(-10, 10) * PIXEL_MULTIPLIER
 
+/obj/item/weapon/reagent_containers/food/drinks/soda_cans/zam_humanhydrator
+	name = "Zam Human Hydrator"
+	desc = "The mothership provides only the best mineral water for humans to drink, REAL minerals included."
+	icon_state = "Zam_HumanHydrator"
+/obj/item/weapon/reagent_containers/food/drinks/soda_cans/zam_humanhydrator/New()
+	..()
+	reagents.add_reagent(WATER, 35)
+	reagents.add_reagent(IRON, 1)
+	reagents.add_reagent(COPPER, 1)
+	reagents.add_reagent(SILVER, 1)
+	reagents.add_reagent(GOLD, 1)
+	reagents.add_reagent(DIAMONDDUST, 1)
+	src.pixel_x = rand(-10, 10) * PIXEL_MULTIPLIER
```

Comment:
no src

Reply:
What do you mean? I'm not sure what I'm supposed to correct. Do I remove the src lines?

Reply:
delete the `src.` in front of pixel_x and pixel_y

Reply:
@Kurfursten Alright, I did it.